### PR TITLE
DM-37868: fix more fallout from DM-31725

### PR DESCRIPTION
**Pull request by TallJimbo**
## Checklist

- [x] ran Jenkins
- [x] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

File: python/lsst/daf/butler/registry/tests/_registry.py [hunk 1]
```diff
@@ -3165,3 +3165,33 @@ def testExposureQueries(self):
...
         )
+
+    def test_long_query_names(self) -> None:
+        """Test that queries involving very long names are handled correctly.
+
+        This is especially important for PostgreSQL, which truncates symbols
+        longer than 64 chars, but it's worth testing for all DBs.
+        """
+        registry = self.makeRegistry()
+        name = "this_is_a_very_long_name_that_exceeds_the_postgresql_sixty_four_char_limit"
```
Comment (andy-slac) [inline python/lsst/daf/butler/registry/tests/_registry.py]: I'm going to believe you, but it's not easy to verify that this is actually longer than 64 chars.

**Review by andy-slac — APPROVED** [review]
Looks good. I can't parse what `login` means in `Remove problematic collection-type login in findDatasets` commit message.

File: python/lsst/daf/butler/registry/tests/_registry.py (hunk 1 above)
Comment (TallJimbo) [reply python/lsst/daf/butler/registry/tests/_registry.py]: I'll replace it.

**Comment by TallJimbo** [comment]
> I can't parse what `login` means

Should have been "logic"; will fix.

**Pull request merged**